tests/scripts/all.sh

```diff
@@ -4922,7 +4922,7 @@ helper_block_cipher_no_decrypt_build_test () {
 
     msg "build: default config + BLOCK_CIPHER_NO_DECRYPT${set_opts:+ + $set_opts}${unset_opts:+ - $unset_opts} with $cflags${ldflags:+, $ldflags}"
     make clean
-    make CC=gcc CFLAGS="$cflags" LDFLAGS="$ldflags"
+    make CFLAGS="-O2 $cflags" LDFLAGS="$ldflags"
```

Comment:
Again, why are we adding `-O2` here? (To be clear, I'm not objecting, it's just that we normally document why a particular set of options is used)